amx/server/natives/a_actors.lua

```diff
@@ -22,27 +23,30 @@ end
 function ApplyActorAnimation(amx, actor, animlib, animname, fDelta, loop, lockx, locky, freeze, time)
 	setPedAnimation(actor, animlib, animname, time, loop, lockx or locky, false, freeze)
 	setPedAnimationSpeed(actor, animname, fDelta)
+	return true
 end
 
 function ClearActorAnimations(amx, actor)
-	setPedAnimation(actor, false)
+	return setPedAnimation(actor, false)
 end
 
 function SetActorFacingAngle(amx, actor, ang)
 	local rotX, rotY, rotZ = getElementRotation(actor) -- get the local players's rotation
-    setElementRotation(actor, rotX, rotY, ang, "default", true) -- turn the player 10 degrees clockwise
+	return setElementRotation(actor, rotX, rotY, ang, "default", true) -- turn the player 10 degrees clockwise
 end
 
 function GetActorFacingAngle(amx, actor, refAng)
 	local rX, rY, rZ = getElementRotation(actor)
 	writeMemFloat(amx, refAng, rZ)
+	return true
 end
 
 function GetActorPos(amx, actor, refX, refY, refZ)
 	local x, y, z = getElementPosition(actor)
 	writeMemFloat(amx, refX, x)
 	writeMemFloat(amx, refY, y)
 	writeMemFloat(amx, refZ, z)
+	return true
```

Comment:
Apparently this method's return behaves the same way `GetActorFacingAngle` does.